pootle/apps/pootle_statistics/models.py

```diff
@@ -25,11 +25,12 @@
 from pootle_store.util import FUZZY, TRANSLATED, UNTRANSLATED
 
 
-EDIT_COEF = 5.0/7
-REVIEW_COEF = 2.0/7
-SUGG_COEF = 0.2
-ANALYZE_COEF = 0.1
-SIMILARITY_THRESHOLD = 0.5
+EDIT_COEF = settings.POOTLE_SCORING_CONF.get('EDIT_COEF', 5.0/7)
```

Comment:
I used `settings.POOTLE_SCORING_CONF[...]` before, but if any user overwrites his settings and miss something he will be able to get errors. I agree that duplication is not good here. So what about making `POOTLE_SCORING_CONF` as optional settings?